packages/forms/src/Components/Concerns/HasFieldWrapper.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Filament\Forms\Components\Concerns;
+
+use Closure;
+
+trait HasFieldWrapper
+{
+    protected string | Closure | null $fieldWrapperView = null;
+
+    public function fieldWrapperView(string | Closure | null $view): static
+    {
+        $this->fieldWrapperView = $view;
+
+        return $this;
+    }
+
+    public function usesInlineWrapper(bool $condition = true): static
```

Comment:
Should this accept a `Closure` too, not sure why you'd want it to but it would be consistent then?